cucumber/api/features/step_definitions/authz_steps.rb

```diff
@@ -82,7 +82,7 @@
 end
 
 Given(/^I create a binary secret value?$/) do
-  @value = Random.new.bytes(16)
+  @value = Random.new.bytes(16).force_encoding(Encoding::UTF_8)
```

Comment:
I don't think they should be compared as strings. I think they should be compared as arrays of bytes.